src/components/Text/Text.stories.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { Text } from "./Text";
+import { withKnobs, text, select } from "@storybook/addon-knobs";
+
+export const Playground = () => {
+  const our_text = text("text", "Default text");
```

Comment:
in JavaScript it's a convention to use camelCase instead of snake_case like in Python

```suggestion
  const ourText = text("text", "Default text");
```